Grbl_Esp32/i2s_ioexpander.cpp

```diff
@@ -345,12 +395,12 @@ int i2s_ioexpander_init(i2s_ioexpander_init_t &init_param) {
 
   I2S0.fifo_conf.dscr_en = 0;
 
-  I2S0.conf_chan.tx_chan_mod = 4; // 0: Dual channel data mode, 4: Mono
```

Comment:
Excellent!  I was wondering about this.  Can we now do 32 bits at 250 kHz?

Reply:
Yes, that's right. I mis-specified the format of the output, so I looked at the technical manual and the actual electrical output and made the right choice.

In debugging, we're going to take the data in "1,2,3..." to make sure that the test data is output in the order it was intended.
Since the same data is output on the right and left channels, the I2S data stream is output as "1,1,2,2,2,3,3,...". (The same data is output on the right and left channels)